WordPress/gutenberg
# test: Improve E2E test consistency

## What?

Improve E2E test environment consistency.

## Why?

Cryptic errors from inconsistent environments are confusing.

## How?

**test: Mitigate conflicts between development server caches**
Occasionally, cryptic npm package dependency errors can occur between
test runs. They appear to originate from stale development caches, e.g.
Metro server. Clearing these caches, which E2E tests should not rely
upon mitigates the likelihood of these errors.

**test: Prefix informative E2E test setup logs**
Merely an attempt to add structure to the outputted logs.

**test: Avoid setup scripts during test runs**
There may be clarity and efficiency by having an explicit setup script.
It was avoid running unnecessary scripts during test runs. It is also
referenced in the current documentation as a script to run
intentionally.

## Testing Instructions




- Verify the `npm run native test:e2e:setup` command succeeds when run.
- Verify the `npm run native test:e2e:ios:local` command does not throw errors.
- Verify the `npm run native test:e2e:android:local` command does not throw
  errors.

### Testing Instructions for Keyboard

n/a

## Screenshots or screencast 
n/a
